src/validation/validatetx.cpp

```diff
@@ -0,0 +1,613 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2015 The Bitcoin Core developers
+// Copyright (c) 2015-2018 The Bitcoin Unlimited developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "validatetx.h"
+
+#include "blockrelay/blockrelay_common.h"
+#include "blockstorage/blockstorage.h"
+#include "blockstorage/sequential_files.h"
+#include "checkpoints.h"
+#include "connmgr.h"
+#include "consensus/consensus.h"
+#include "consensus/merkle.h"
+#include "consensus/tx_verify.h"
+#include "dosman.h"
+#include "expedited.h"
+#include "init.h"
+#include "requestManager.h"
+#include "sync.h"
+#include "timedata.h"
+#include "txadmission.h"
+#include "txorphanpool.h"
+#include "ui_interface.h"
+#include "validationinterface.h"
+
+#include "txmempool.h"
+
+#include "validation.h"
+
```

Comment:
2 things on the header above: 

- sorry for the sloppy comment, but do you we really need all the above includes? (e.g. `expedited.h`)
- why we have empty lines between the last two headers include?

Reply:
ill fix the spacing and see if i can remove some of the headers